crypto/keys/keybase.go

```diff
@@ -197,6 +205,15 @@ func (kb dbKeybase) Get(name string) (Info, error) {
 	return readInfo(bs)
 }
 
+func (kb dbKeybase) GetByAddress(address types.AccAddress) (Info, error) {
+	ik := kb.db.Get(addrKey(address))
+	if len(ik) == 0 {
+		return nil, fmt.Errorf("key with address %s not found", address.String())
```

Comment:
No need to actually use `String()` here as Go will automatically do this for you (if the type implements `Stringer`) 👍